include/circt/Dialect/SV/SVGenerate.td

```diff
@@ -50,3 +50,26 @@ def GenerateCaseOp : SVOp<"generate.case",
 
   let hasVerifier = true;
 }
+
+def ForProceduralOp : SVOp<"for.procedural", [SingleBlock, NoTerminator,
+                        ProceduralOp, ProceduralRegion]> {
+  let summary = "System verilog for loop";
+
+  let description = [{
+    This operation defines System Verilog for loop.
+  }];
+
+  let regions = (region SizedRegion<1>:$body);
+  let arguments = (ins I32Attr:$lowerBound,
```

Comment:
Attributes or values?

Reply:
Thanks, I think it makes sense to use normal values as lower and upper values so values are used in the latest version